docs/DeveloperGuide.md

```diff
@@ -497,6 +547,23 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Viewing a person
+
+1. Viewing a person while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+
+    1. Test case: `view 1`<br>
+       Expected: First person is viewed from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
Not sure why there is mention of a deleted contact here, is a person ever deleted in this use case?